.github/workflows/main.yml

```diff
@@ -19,11 +20,245 @@ on:
     - master
 
 env:
-  IMG_CORE: docker.pkg.github.com/kotekan/kotekan/kotekan-core
-  IMG_IWYU: docker.pkg.github.com/kotekan/kotekan/kotekan-iwyu
+  REGISTRY: docker.pkg.github.com
+  IMG_CORE: "docker.pkg.github.com/kotekan/kotekan/kotekan-core"
+  IMG_IWYU: "docker.pkg.github.com/kotekan/kotekan/kotekan-iwyu"
+  IMG_CI: "docker.pkg.github.com/${{ github.repository }}/kotekan-ci"
```

Comment:
We likely need to update the naming here to include the Ubuntu versions.  Otherwise I'm not sure what the difference between `kotekan-core` and `kotekan-ci` is in terms of base Ubuntu version.